database/schema.cql

```diff
@@ -1,67 +1,37 @@
+DROP KEYSPACE IF EXISTS bnb;
 CREATE KEYSPACE IF NOT EXISTS bnb
 WITH replication = {
   'class' : 'NetworkTopologyStrategy',
   'datacenter1' : 1
   };
 
---not sure if still necessary since query driven. But when log in maybe?
-CREATE TABLE bnb.users (
-  user_id uuid,
-  email varchar,
-  first_name varchar,
-  last_name varchar,
-  --saved_lists list <listName>,  lists should just be in here as names?
-  PRIMARY KEY (user_id)
-)
+USE bnb;
 
---not sure if necessary since query driven.
-CREATE TABLE bnb.rooms (
-  room_id uuid,
-  user_id uuid,
+--photo order
+CREATE TABLE bnb.photos_by_room (
+  room_id int,
+  photo_order int,
   title varchar,
   address varchar,
-  --room_photos set<image> should just be a bunch of images in here
-  PRIMARY KEY (room_id)
-)
-
---as if it was owner, would then lead to bnb.rooms by room id?
-CREATE TABLE bnb.rooms_by_user (
-  user_id uuid,
-  room_id uuid,
-  titel varchar,
-  address varchar,
-  PRIMARY KEY (user_id)
-)
-
--- do i need image id column
-CREATE TABLE bnb.photos_by_room (
-  room_id uuid,
-  image_url varchar, -- should just be a bunch of images in here?
+  image_url varchar,
   image_description text,
-  PRIMARY KEY (room_id)
-)
-
---this doesn't seem necessary
--- CREATE TABLE bnb.rooms_by_address (
---   address varchar,
---   user_id uuid,
---   title varchar,
---   room_photos set<image>,
---   PRIMARY KEY (address)
--- )
+  PRIMARY KEY (room_id, photo_order) --add photo_order as composite so room_id and photo_order combo will be unique
+) WITH compaction = { 'class': 'org.apache.cassandra.db.compaction.LeveledCompactionStrategy'};
 
 CREATE TABLE bnb.list_by_user (
-  user_id uuid,
-  list_id uuid,
+  user_id int,
+  list_id int,
   list_name varchar,
-  PRIMARY KEY(user_id, list_id)
-)
+  room_id int,
+  PRIMARY KEY(user_id, list_id) --use composite so user_id and list_id combo will be unique
```

Comment:
About composite: and what if user doesn't have any lists, it can be a possible problem for hashing and then reading from table, just to think about it.